src/main/java/creativei/manager/impl/InquiryManagerImpl.java

```diff
@@ -136,9 +134,27 @@ public ResponseObject getUnattendedInquiry(String boolParam,String statusParam)
         List<InquiryVo> inquiryVos=ResponseHelper.getUnattendedInquiryResponse(inquiryService.getUnattendedInquiry(boolParam,statusParam));
         return ResponseObject.getResponse(inquiryVos);
     }
+
+    @Override
+    public ResponseObject getInquiryCount() {
```

Comment:
Too many database requests for this request. Optimize queries to reduce db requests